### ISSUE TITLE
Microsoft .NET Core 3.1.0 Windiws Server Hosting installer installs .Net Core Runtime 3.1.0 Preview 3

### ISSUE THREAD

**Issue by msschl** [dotnet/runtime#789]
I have downloaded and installed the latest [Hosting Bundle 3.1.0-win](https://download.visualstudio.microsoft.com/download/pr/fa3f472e-f47f-4ef5-8242-d3438dd59b42/9b2d9d4eecb33fe98060fd2a2cb01dcd/dotnet-hosting-3.1.0-win.exe). But the installer installs the .Net Core Runtime - 3.1.0 Preview 3 (x64) and (x86)

![image](https://user-images.githubusercontent.com/5054780/70715329-5ad42900-1cea-11ea-9699-6146b34b8fa7.png)

**Comment by OneCyrus** [comment]
same issue here. there seems to be missing files in the NETCore.App directory. Just one file while there are over 200 for every other version.

![image](https://user-images.githubusercontent.com/7747800/70721363-b48e2080-1cf5-11ea-91f6-6fe3e392a0b1.png)

**Comment by msschl** [comment]
I have enabled stdout, looked into the logs and found the reason for the "HTTP Error 500.31 - ANCM Failed to Find Native Dependencies" error:

```
Error:
  An assembly specified in the application dependencies manifest (AppX.deps.json) was not found:
    package: 'System.IO.Compression', version: '4.3.0'
    path: 'runtimes/win/lib/netstandard1.3/System.IO.Compression.dll'
```

So the installation of the 3.1.0 Preview 3 (x64) is not the reason for the HTTP Error 500.31

**Comment by msschl** [comment]
Solved the following error by supplying the `--no-self-contained` flag to the `dotnet publish` command:

```
Error:
  An assembly specified in the application dependencies manifest (AppX.deps.json) was not found:
    package: 'System.IO.Compression', version: '4.3.0'
    path: 'runtimes/win/lib/netstandard1.3/System.IO.Compression.dll'
```

**Comment by msschl** [comment]
But I am still not able to update to 3.1 since the Hosting Bundle Installer installs .net core 3.1 preview 3

**Comment by OneCyrus** [comment]
i installed the runtime as well now and it fixed the issue for me

**Comment by blogcraft** [comment]
Having the sale problem with Hosting bundle

**Comment by msschl** [comment]
I’ve updated the issue. The error was not related to the installation.

But there is still the issue with the hosting bundle. It installs 3.1.0 Preview 3

When will this be fixed?

**Comment by msschl** [comment]
Hosting bundle is fixed for version 3.1.1.

**Comment by dagood** [comment]
Sorry I missed this issue back when it was reported! Both the preview branding issue and missing files issues described here were reported elsewhere and documented as known issues for 3.1.0 at https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1-known-issues.md#aspnet-core, with a workaround for the broken install one. They're indeed both resolved with 3.1.1.
